charts/dapr/values.yaml

```diff
@@ -35,6 +35,9 @@ global:
   prometheus:
     enabled: true
     port: 9090
+  metrics: {}
```

Comment:
This needs the `{}` removed

Reply:
was following same pattern as later in this file like
```
  extraVolumes: {}
  #  sentry:
  #  placement:
  #  operator:
  #  injector:
```

Reply:
I don't believe that's valid YAML syntax

Reply:
it is, an empty map in yaml, I can remove it but will have to change the others for consistency

Reply:
```
metrics: {}
```

is valid

... but if you uncomment the line below:

```
metrics: {}
   http:
```

is not

Reply:
this is what I mean of consistency:
```
  metrics: {}
  # http:
  #   increasedCardinality: false # set to true to revert to old behavior by default (dapr < 1.14)
  mtls:
    enabled: true
    workloadCertTTL: 24h
    allowedClockSkew: 15m
    controlPlaneTrustDomain: "cluster.local"
    # If set to true, a bound service account token will be mounted and used to
    # authenticate to Sentry.
    mountSentryVolume: true
    # Used to override `dapr-sentry.{{ .Release.Namespace }}.svc.cluster.local:443`
    #sentryAddress:
  # extraVolumes and extraVolumeMounts are used to mount additional volumes to
  # the Dapr control plane pods. Useful for using alternative authentication
  # credentials to sentry.
  extraVolumes: {}
  #  sentry:
  #  placement:
  #  operator:
  #  injector:
  extraVolumeMounts: {}
  #  sentry:
  #  placement:
  #  operator:
  #  injector:
```

Reply:
@ItalyPaleAle actually this does not work and will lead to npe :
```
helm template dapr
Error: template: dapr/charts/dapr_config/templates/dapr_default_config.yaml:18:15: executing "dapr/charts/dapr_config/templates/dapr_default_config.yaml" at <.Values.global.metrics.http>: nil pointer evaluating interface {}.http
```

Reply:
What doesn't work?

Reply:
removing the `{}`

Reply:
Yes please do that.

Every other option is invalid

```yaml
# This is invalid
metrics:

# This is invalid too
metrics: {}
  http: ...
```